Makefile

```diff
@@ -0,0 +1,40 @@
+#
+#
+# $Id$
+#
+.PHONY: 2to3 coverage docs init install lint test
+
+# TODO: Change from stub to working code
+2to3:
```

Comment:
This needs a comment to explain what it does.   I really don't know what 2to3 could mean.